migrations/Version202411071618031101_taoQtiItem.php

```diff
@@ -6,27 +6,27 @@
 
 use Doctrine\DBAL\Schema\Schema;
 use oat\tao\scripts\tools\migrations\AbstractMigration;
-use oat\taoQtiItem\scripts\install\AddLabelInjectorForExport;
+use oat\taoQtiItem\scripts\install\RemoveLabelInjectorFromExport;
 
 /**
- * Auto-generated Migration: Please modify to your needs!
+ * Remove LabelBasedLomOntologyClassificationExtractor if present
  *
  * phpcs:disable Squiz.Classes.ValidClassName
  */
-final class Version202407091133271101_taoQtiItem extends AbstractMigration
+final class Version202411071618031101_taoQtiItem extends AbstractMigration
 {
     public function getDescription(): string
     {
-        return '';
+        return 'Remove oat\taoQtiItem\model\qti\metadata\ontology\LabelBasedLomOntologyClassificationExtractor' .
+            ' if present';
     }
 
     public function up(Schema $schema): void
     {
-        $this->runAction(new AddLabelInjectorForExport());
+        $this->runAction(new RemoveLabelInjectorFromExport());
```

Comment:
We are renaming the class and keeping the same migration, it will break previous installation using the old injector

Reply:
Version202411071618031101_taoQtiItem migration name also new

Reply:
IDK why its shows it as rename. Its new file generated via migration tool.